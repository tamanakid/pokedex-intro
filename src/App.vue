<template>
  <div id="app">
    <MainLayout />
  </div>
</template>

<script>
import MainLayout from './modules/MainLayout.vue'

export default {
  name: 'app',
  components: {
    MainLayout
  }
}
</script>

<style lang="scss">
body {
  background-color: $poke-black-bg;
}

#app {
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;

  color: $poke-scarlett-light;
  font-family: $font-family-main, 'Avenir', sans-serif;
  text-align: center;
}
</style>
