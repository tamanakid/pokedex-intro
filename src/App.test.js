import App from './App.vue'
import { shallow } from 'vue-test-utils'

describe("App", () => {

    let shallowComponent = shallow(App);

    it("Has expected HTML structure", () => {
        expect(shallowComponent.element).toMatchSnapshot();
    });

});