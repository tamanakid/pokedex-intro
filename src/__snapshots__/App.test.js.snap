// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Has expected HTML structure 1`] = `
<div
  id="app"
>
  <!---->
</div>
`;
